src/scripts/generate-imported-targets-from-snyk.ts

```diff
@@ -41,10 +41,19 @@ export function projectToTarget(
   };
 }
 
+export function imageToTarget(
+  project: Pick<SnykProject, 'name'>,
+): Target {
+  return {
+    name: project.name
```

Comment:
could you add a quick simple test  `test/scripts/generate-imported-targets-from-snyk.test.ts` so we don't accidentally ever break it. Using a real project name from Snyk as an example?

Reply:
Example test: https://github.com/snyk-tech-services/snyk-api-import/pull/170/files#diff-18c13dd74a3e99d752495aebb709c5ae0652f2ac1e6ec6f364dd3d35c37ebc06R59-R70